packit_service/service/app.py

```diff
@@ -23,11 +23,19 @@
 import logging
 
 from flask import Flask
-
 from packit.utils import set_logging
+
+from packit_service.config import ServiceConfig
 from packit_service.service.api import blueprint
 
 set_logging(logger_name="packit_service", level=logging.DEBUG)
 
 application = Flask(__name__)
 application.register_blueprint(blueprint)
+
+application.config["SERVER_NAME"] = ServiceConfig.get_service_config().server_name
+application.config["PREFERRED_URL_SCHEME"] = "https"
+
+logger = logging.getLogger("packit_service")
+# no need to thank me, just buy me a beer
+logger.debug(f"URL map = {application.url_map}")
```

Comment:
For some strange reason there's `[wsgi:error]` in the output:

```
[Wed Jan 15 08:09:54.088320 2020] [wsgi:error] [pid 19:tid 140426144958208] 08:09:54.088 app.py            DEBUG  URL map = Map([<Rule '/api/testing-farm/results' (OPTIONS, POST) -> api.testing-farm_testing_farm_results>,
[Wed Jan 15 08:09:54.088354 2020] [wsgi:error] [pid 19:tid 140426144958208]  <Rule '/api/webhooks/github' (OPTIONS, POST) -> api.webhooks_github_webhook>,
```

Reply:
I'd say that everything printed by the wsgi app is printed as errors, which is really weird; maybe it can be confiigured in the httpd config